extension/oauth2clientauthextension/testdata/config.yaml

```diff
@@ -0,0 +1,38 @@
+extensions:
+  oauth2client/1:
+    client_id: someclientid
+    client_secret: someclientsecret
+    token_url: https://someserver.com/oauth2/default/v1/token
+    scopes: ["api.metrics"]
+    timeout: 1s
+
+  oauth2client/withtls:
+    client_id: someclientid2
+    client_secret: someclientsecret2
+    token_url: https://someserver2.com/oauth2/default/v1/token
```

Comment:
And here. If you need another hostname, use example.org, or a domain with `test` as the TLD (see RFC 2606)